Clinical-Genomics/scout
# Update igv.js to 2.4.7 with fix on genes track

This PR updated IGV.js from version 2.7.0 to 2.7.4 and removes the visibility window parameter in gene track that caused this following [bug](https://github.com/Clinical-Genomics/scout/issues/2323).

See the thread with a developer of igv.js [here](https://github.com/igvteam/igv.js/issues/1261)

**How to test**:
1. Install the branch on stage, and go to this variant: https://scout-stage.scilifelab.se/cust002/F0013248-3/17981289e34c2c706ecf53f8de78b0d1
2. Click on the igv button and then look in igv for this gene interval: chr14:55,369,194-55,369,443
3. The gene "GCH1" should be displayed directly in the gene track

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
